packages/node/src/providers/initialize.test.ts

```diff
@@ -41,16 +43,35 @@ const chains: ChainConfig[] = [
     contracts: {
       AirnodeRrp: '0x9AF16dE521f41B0e0E70A4f26F9E0C73D757Bd81',
     },
-    id: 3,
-    providers: [{ name: 'infura-ropsten', url: 'https://ropsten.infura.io/v3/<key>' }],
+    id: '3',
+    providerNames: [chainProviderName3],
     type: 'evm',
   },
 ];
 
+const environmentConfig: EnvironmentConfig = {
+  securitySchemes: [],
+  chainProviders: [
+    {
+      chainType: 'evm',
+      chainId: '1',
+      name: chainProviderName1,
+      envName: `cp_evm_1_${chainProviderName1}`,
+    },
+    {
+      chainType: 'evm',
+      chainId: '3',
+      name: chainProviderName3,
+      envName: `cp_evm_3_${chainProviderName3}`,
+    },
+  ],
+};
+
 describe('initializeProviders', () => {
   it('sets the initial state for each provider', async () => {
-    const nodeSettings = fixtures.buildNodeSettings({ chains });
-    const config = fixtures.buildConfig({ nodeSettings });
+    const config = fixtures.buildConfig({ chains, environment: environmentConfig });
+    process.env[config.environment.chainProviders[0].envName] = 'https://mainnet.infura.io/v3/<key>';
```

Comment:
These probably persist between tests right? I wonder if we should delete them at the end of the test.

Reply:
I think we would have needed to worry about this if it was the node itself adding/removing environment variables

Reply:
It will persist during the test. You can clean in the beforeEach to make sure you don't have an issue there